mlir/include/mlir/Dialect/MIOpen/LowerMIOpenOps.h

```diff
@@ -8416,6 +9496,7 @@ struct ThreadwiseCopyRewritePattern
       ArrayAttr layeredSourceTransformMetadata;
       ArrayAttr layeredDestTransformMetadata;
 
+      legacyLoad = true;
```

Comment:
Please explain this

Reply:
This logic has been eliminated in the latest logic.